cmd/go-camo/main.go

```diff
@@ -72,6 +75,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	// set the server name
+	ServerName := opts.ServerName
+	if ServerCommitSha != "" {
+		ServerName = fmt.Sprintf("%s (%s)", opts.ServerName, ServerCommitSha)
```

Comment:
I think this needs to be gated behind a cli flag.
People are often uncomfortable exposing versions of the software they are running (even nginx has an option to hide the version).

I'm also not sure how much merit (to the go-camo project) including the git has has, vs a more classical version string.

Reply:
Sure, happy to chuck this behind a flag 👍

Reply:
Note that for a non-release version, the git-sha is also included in server ver.

```
go-camo(master)$ ./build/bin/go-camo -V
go-camo 1.0.18-4-ge81b1d3 (go1.10.3,gc-amd64)
go-camo(master)$ ./build/bin/go-camo -V | awk '{print $2}'
1.0.18-4-ge81b1d3
```

You could easily add that to a startup server name flag invocation:

```
$ go-camo --server-name="$(go-camo -V | awk '{print $2}')" -k ....
```

Reply:
I did initially consider this but didn't need the extra bits of the golang version so just thought it would be better to separate out these responsibilities instead of mangling the existing data.

Reply:
Addressed the flag in 4a6c128